src/ContentValidationListenerFactory.php

```diff
@@ -10,12 +10,14 @@
 use Zend\ServiceManager\FactoryInterface;
 use Zend\ServiceManager\ServiceLocatorInterface;
 
-class ContentValidationListenerFactory
+class ContentValidationListenerFactory implements FactoryInterface
```

Comment:
Thanks for you feedback, we noticed that! you can check this thread : https://github.com/zfcampus/zf-content-validation/pull/76#discussion_r71698040